src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp

```diff
@@ -482,7 +490,7 @@ class npc_hellfire_sentry : public CreatureScript
                 Initialize();
             }
 
-            void JustEngagedWith(Unit* /*who*/) override { }
+            void JustEngagedWith(Unit* who) override { }
```

Comment:
this will throw a warning about unused who variable

Reply:
I didn't see any when i built it.

Reply:
>[ 60%] Building CXX object src/server/scripts/CMakeFiles/scripts_outland.dir/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp.o
/home/circleci/project/src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp:95:40: fatal error: unused parameter 'who' [-Wunused-parameter]
            void JustEngagedWith(Unit* who) override { }

from https://circleci.com/gh/TrinityCore/TrinityCore/5683?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
OK .. Left those the same.